src/theme/components/button.ts

```diff
@@ -3,19 +3,82 @@ import { defineStyleConfig } from "@chakra-ui/react";
 export const buttonTheme = defineStyleConfig({
   baseStyle: {
     backgroundColor: "primary.600",
+    borderRadius: "base",
+    // token?
     color: "white",
```

Comment:
Referencing the `white` token in `/tokens/colors.ts` is fine here. It just looks weird because `white` also happens to be a named color in regular css. When Chakra sees a token and built-in css value of the same name, the token "wins".